app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -67,6 +67,8 @@ export default Controller.extend(bufferedProperty("model"), {
   username_filters: null,
   filter: null,
   quoteState: null,
+  _retryRateLimited: false,
```

Comment:
same a my other comment, if it's not meant to be computed, you could just define it in init, although it's not an array or object, so it's fine like this.